app/questionnaire/placeholder_parser.py

```diff
@@ -46,9 +47,16 @@ def _resolve_value_source(self, value_source):
         if value_source["source"] == "metadata":
             return self._resolve_metadata_value(value_source["identifier"])
         if value_source["source"] == "list":
-            if value_source.get("id_selector") == "same_name_items":
-                return self._list_store[value_source["identifier"]].same_name_items
-            return len(self._list_store[value_source["identifier"]])
+            id_selector = value_source.get("id_selector")
+            list_store: ListModel = self._list_store[value_source["identifier"]]
+
+            if id_selector == "same_name_items":
+                return list_store.same_name_items
+
+            if id_selector == "first":
+                return list_store.first
```

Comment:
```suggestion
            if id_selector:
                return getattr(list_store, id_selector)
```

Would ☝️ be more elegant?